erdpy/docker.py

```diff
@@ -0,0 +1,57 @@
+import os
+import logging
+import subprocess
+from pathlib import Path
+from typing import List
+from erdpy.errors import DockerMissingError
+
+
+logger = logging.getLogger("build-with-docker")
+
+
+def is_docker_installed():
+    try:
+        result = subprocess.run(["docker", "--version"], capture_output=True, text=True)
```

Comment:
Actually, I think this will raise an exception if docker is not installed.

Can you make a (manual) test, say, using `["dummy-test-docker", "--version"]`?

Also, `subprocess.check_output()` can be used, perhaps?

Reply:
Actually, it doesn't raise an error, it says "dummy-test-docker: command not found". I expect `subprocess.check_output` to have the same behaviour, but it's recomended to use `subprocess.run`